src/index.html

```diff
@@ -11,7 +11,98 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"       method="post">
+      <fieldset class="wrapper" >
+        <legend>Personal information</legend>
+        <label class="inputs">
+          Surname:
+          <input type="text"
```

Comment:
```suggestion
          <input 
            type="text"
```
start first attribute on a new line (checklist p3), please fix this everywhere